### ISSUE TITLE
Test Run occasionally crashes AppKit by running off the main thread

### ISSUE THREAD

**Issue by chrisballinger** [vitor251093/wineskin#29]
I don't have a stack trace handy, but it has happened to me a number of times.

**Comment by Gcenx** [comment]
I'm also getting the same;

I think it's related to when I added checking for XQuart.
As I have it removed in my local development branch and I have not crashed since, that could just be luck on my end

**Comment by vitor251093** [comment]
No, I noticed that early. Check here:
https://github.com/vitor251093/wineskin/blob/master/WineskinApp/WineskinAppDelegate.m

Like 139. Commenting the specified lines make the crash stop.

**Comment by Gcenx** [comment]
Yeah I seen that note, but when I removed all XQuartz related items on my end I had no crashes since, properly just my luck like I said ;P

**Comment by vitor251093** [comment]
Maybe xD

But it may be a separated issue as well. Let's keep an eye opened to that.
